framework/src/application/node/forger/forger.js

```diff
@@ -124,11 +122,11 @@ class Forger {
 			throw new Error('Invalid password and public key combination');
 		}
 
-		const filters = {
-			address: getAddressFromPublicKey(keypair.publicKey.toString('hex')),
-		};
+		const address = getAddressFromPublicKey(keypair.publicKey.toString('hex'));
 
-		const [account] = await this.storage.entities.Account.get(filters);
+		const account = await this.blocksModule.dataAccess.getAccountByAddress(
```

Comment:
can we not use publicKey to get the account? and publicKey is also indexed
```
const account = await this.blocksModule.dataAccess.getAccountsByPublicKey([keypair.publicKey.toString('hex')])
```

The reason is we could avoid an extra call to get the address first `getAddressFromPublicKey` from above line and directly use `getAccountsByPublicKey`.